lib/gi/lce/configuration.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'gi/configuration'
+
+module GI
+  module Lce
+    class Configuration < GI::Configuration
+      def use_mocks?
+        Settings.gids.lce.use_mocks || false
```

Comment:
I see this is defined in [values.yaml staging](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/staging/values.yaml#L2128-L2129), but not in other envs. As the default, please add this configuration to config/settings.yml (in vets-api). The value in the manifest repo will override the default you set in settings.yml.